### PR TITLE
Provisions for possible emergency reimbursement

### PR DESCRIPTION
**Pull request by asreich**
Add couple of lines outlining reimbursement cases

### CODE REVIEW

File: board_orders.md [hunk 1]
```diff
@@ -89,7 +89,9 @@ The overdue balance includes, but is not limited to, dues, fees, and merchandise
 **Article IV. Payment and Refunds**
 
 * Membership dues must be paid via a payment method sanctioned by the Treasurer.
-* Initiated members are not eligible for any form of refund.
+* If an initiated member who has already paid dues, decides before the Treasurer-specific dues date at midnight (CDT) that they for some reason no longer desire to be active, they are eligble for a full refund of what they have paid up till that date.
```
Comment (ThomasGaubert) [inline board_orders.md]: Probably better worded as:

>Initiated members are eligible for a full refund of any paid membership dues less any third party processing fees until midnight (CDT) of the actives-specific dues date.

File: board_orders.md [hunk 2]
```diff
@@ -89,7 +89,9 @@ The overdue balance includes, but is not limited to, dues, fees, and merchandise
 **Article IV. Payment and Refunds**
 
 * Membership dues must be paid via a payment method sanctioned by the Treasurer.
-* Initiated members are not eligible for any form of refund.
+* If an initiated member who has already paid dues, decides before the Treasurer-specific dues date at midnight (CDT) that they for some reason no longer desire to be active, they are eligble for a full refund of what they have paid up till that date.
+* If an initiated member decides after the Treasurer-specfied dues date that they no longer desire to be active, they are still responsible for paying any remaining dues balance.
```
Comment (ThomasGaubert) [inline board_orders.md]: >Initiated members are responsible for any remaining dues balance if a refund is not requested by the deadline defined above.

File: board_orders.md [hunk 3]
```diff
@@ -89,7 +89,9 @@ The overdue balance includes, but is not limited to, dues, fees, and merchandise
 **Article IV. Payment and Refunds**
 
 * Membership dues must be paid via a payment method sanctioned by the Treasurer.
-* Initiated members are not eligible for any form of refund.
+* If an initiated member who has already paid dues, decides before the Treasurer-specific dues date at midnight (CDT) that they for some reason no longer desire to be active, they are eligble for a full refund of what they have paid up till that date.
+* If an initiated member decides after the Treasurer-specfied dues date that they no longer desire to be active, they are still responsible for paying any remaining dues balance.
+* If an intiated member decides after the Treasurer-specified dues date that they can no longer be active due to a personal, financial, family, or other such emergency, they can be granted a refund by unanimous vote of the Board.
```
Comment (ThomasGaubert) [inline board_orders.md]: >In the event of a personal, financial, family, or other such emergency after the deadline specified above, the Board may unanimously vote to grant a full refund less any third party processing fees.

File: board_orders.md (hunk 1 above)
Comment (elmdecoste) [reply board_orders.md]: Initiation happens at the end of the semester and it seems like after that point there should not be reimbursement

**Comment by ThomasGaubert** [comment]
Approved at December 4th OM.

**Pull request merged**